qa/workunits/rados/test_rados_tool.sh

```diff
@@ -381,6 +381,11 @@ test_ls() {
     then
         die "Created $OBJS objects in default namespace but saw $CHECK"
     fi
+    CHECK=$("$RADOS_TOOL" -p $p ls --regex obj.* 2> /dev/null | wc -l)
```

Comment:
please quote `obj.*` like `"obj.*"`, to prevent the shell from expanding it. also, better off specifying a pattern other than `obj.*` like `obj1.*` and update the `if` clause accordingly, so this feature can be tested in a more discriminative way.